python/pyarrow/tests/test_table.py

```diff
@@ -117,6 +117,15 @@ def test_combine_chunks():
     expected = pa.array([1, 2, 1, 2])
     assert res.equals(expected)
 
+def test_chunked_array_can_combine_chunks_with_no_chunks():
```

Comment:
Add newline to fix Python lint.
```suggestion

def test_chunked_array_can_combine_chunks_with_no_chunks():
```